### DISCUSSION TITLE
2 Relation endpoints seem to have [Oct 11]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6734] [Discord Threads]
> 🙋‍♂️ **vadminas**  
## 2. Relation endpoints seem to have disappeared in swagger for new DBs. I understand this might be realted to introduction of links. But how to create relations over API then?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1149658443750314034/image.png?ex=653b965c&is=6529215c&hm=e05ed81122c540c0b68a7ee347e76de6eeb14cf6b1eccf396814bd6904c5b56d&)

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### We introduced Links type so probably for links column it may missing the api entries, will check it and let you know  
#  
> 👨‍💻 **pranavxc**  
#### Looks like that's the problem  
#  
> 👨‍💻 **pranavxc**  
#### Creating a bug fix PR it will be fine in next release  
#  
> 🙋‍♂️ **vadminas**  
#### Will support for current form of relations (for existing DBs) be dropped in future versions? I.e. if I have DBs with previous form of relation (non-link type), will they be supported indefinitely, or I better migrate to links right now?  
#  
> 👨‍💻 **pranavxc**  
#### We are not planning to drop support for the existing LTAR at the moment... only thing is we are not providing an option to create new LTAR since it will be slower  
#  
> 👨‍💻 **pranavxc**  
#### Just discussed with the team, in future we may remove it and migrate existing column to Links  
#  
> 🙋‍♂️ **vadminas**  
#### Found one glitch in the fix related to new links functionality though. The  endpoints for setting relations appeared but column name comes up empty in swagger gui - so still impossible to set relation through swagger page.  
![image0](https://cdn.discordapp.com/attachments/1149658444010369126/1150472612267434065/image.png?ex=653e8c9d&is=652c179d&hm=e4c2448150b6cff6ab9f0252ad3f7cd028683581e512786d8995c947987eb7d2&)  
#  
> 👨‍💻 **pranavxc**  
#### Oh I will check that one and thanks for pointing out  
#  
> 👨‍💻 **pranavxc**  
#### tomorrow we are planning a release with 2 minor bugs will include with that  
#  
> 👨‍💻 **pranavxc**  
#### Fixed here - https://github.com/nocodb/nocodb/pull/6349  
#  
> 🙋‍♂️ **vadminas**  
#### Oh my gosh, are you serious? 😄  
#  
> 🙋‍♂️ **vadminas**  
#### You are too fast, guys! 🙂  
#  
> 👨‍💻 **pranavxc**  
#### Will share the pr build soon to confirm  
#  
> 👨‍💻 **pranavxc**  
#### https://github.com/nocodb/nocodb/actions/runs/6142391130  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1149658444010369126)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
